apps/storybook/src/Selection.stories.tsx

```diff
@@ -106,10 +154,46 @@ ChangeSelectionColors.argTypes = {
   },
 };
 
+export const PersistSelection: Story<Exclude<TemplateProps, 'children'>> = (
+  args
+) => {
+  const { selectionType, color } = args;
+
+  return (
+    <Template {...args}>
+      {(selection) =>
+        selection ? (
+          <SelectionComponent
+            selectionType={selectionType}
+            color={color}
+            {...selection}
+          />
+        ) : null
+      }
+    </Template>
+  );
+};
```

Comment:
Not super happy about how this turned out. If you have a better idea...

Also, it seems there is a flickering of the selection component when releasing the mouse

Reply:
Yeah, I don't think the render prop pattern is needed here -- it's just confusing. You can move the `childrenSelection` state here (and rename it to `savedSelection` or just `selection`), forward an `onSelectionEnd` prop through the story template, and just render the selection as a simple child of the template.

Reply:
Implemented this. I just had to put `onSelectionStart` as well to reset the saved selection